mm/mixtures.py

```diff
@@ -10,7 +10,7 @@
 import shutil
 import struct
 import tempfile
-
+import stat
```

Comment:
Check the contributing guidelines: there should be three blocks of imports
1. From other recipes
2. From standard libraries
3. From other libraries (including Sisyphus)
each block sorted alphabetically. 

I think stat belongs to the standard libraries and should therefore be inserted between shutil and struct. And we keep the empty line separating sisyphus.

Reply:
Ah my bad, will read the guidelines more carefully next time.